### DISCUSSION TITLE
支持免登录获取用户信息

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [SPlayer-Dev/SPlayer#272] [想法 Ideas]
最近的封号事件让很多用户在一段时间内都无法登录，获取不到歌单，十分不方便。

是否能通过搜索用户名或id，获取用户的公开信息，达到类似登录的效果呢？

YesPlayMusic曾在[网页端（第三方部署）](https://my-yesplaymusic-five.vercel.app)实现了这个功能，可以获取到所有公开歌单。

不是很了解NeteaseCloudMusicApi，所以不知道是否还能拿到更多数据，但能访问公开歌单就已经方便很多了。

**Comment by imsyy** [comment]
可以尝试下使用 UID 模式登录，就是云盘用不了了

**Reply by FrzMtrsprt** [reply]
感谢光速开发
